### DISCUSSION TITLE
pinia @nuxtjs/tailwindcss Conflicting peer dependency

### DISCUSSION THREAD

**Discussion by sduncan-mbt** [nuxt/nuxt#16614] [General]
Wasn't sure where to go with this so thought I would start it off in here.

when I run...

npx nuxi init test-app
cd test-app
npm i pinia
npm install --save-dev @nuxtjs/tailwindcss

everything goes fine but if I do...

npx nuxi init test-app
cd test-app
npm install --save-dev @nuxtjs/tailwindcss
npm i pinia

I get....

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: vue@3.2.40
npm ERR! node_modules/vue
npm ERR!   peer vue@"^3.2.39" from @nuxt/vite-builder@3.0.0-rc.11
npm ERR!   node_modules/@nuxt/vite-builder
npm ERR!     @nuxt/vite-builder@"3.0.0-rc.11" from nuxt@3.0.0-rc.11
npm ERR!     node_modules/nuxt
npm ERR!       dev nuxt@"3.0.0-rc.11" from the root project
npm ERR!   peer vue@"^3.2.25" from @vitejs/plugin-vue@3.1.2
npm ERR!   node_modules/@vitejs/plugin-vue
npm ERR!     @vitejs/plugin-vue@"^3.1.0" from @nuxt/vite-builder@3.0.0-rc.11
npm ERR!     node_modules/@nuxt/vite-builder
npm ERR!       @nuxt/vite-builder@"3.0.0-rc.11" from nuxt@3.0.0-rc.11
npm ERR!       node_modules/nuxt
npm ERR!         dev nuxt@"3.0.0-rc.11" from the root project
npm ERR!   7 more (@vitejs/plugin-vue-jsx, @vue/server-renderer, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! pinia@"*" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vue@2.6.14
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 2.7" from @vue/composition-api@1.7.1
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.4.0" from pinia@2.0.23
npm ERR!     node_modules/pinia
npm ERR!       pinia@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

anyone who knows more than me got an suggestion?

**Comment by ProxyAi** [comment]
If you could provide your package.json might of helped#, i made this repo [Nuxt 3 Minimal Starter + tailwind + pinia + i18n](https://github.com/xorenio/nuxt3-start)
